src/lightning/pytorch/trainer/trainer.py

```diff
@@ -1070,6 +1070,14 @@ def __setup_profiler(self) -> None:
         self.profiler._lightning_module = proxy(self.lightning_module)
         self.profiler.setup(stage=self.state.fn, local_rank=local_rank, log_dir=self.log_dir)
 
+    def print(self, *args: Any, **kwargs: Any) -> None:
+        """Print something only on the first process.
```

Comment:
I see we are not using "first process" in other docstrings.
This is not global rank zero, but local rank zero, so clarifying expectations for the user would help.